CHANGELOG.md

```diff
@@ -3,71 +3,129 @@
+
+
+### Experimental
+
+- LCOW: Add `--platform` to `docker import` [docker/cli#1375](https://github.com/docker/cli/pull/1375) / [docker/cli#1371](https://github.com/docker/cli/pull/1371)
+- LCOW: Add LinuxMetadata support by default on Windows [moby/moby#37514](https://github.com/moby/moby/pull/37514)
+- LCOW: Mount to short container paths to avoid command-line length limit [moby/moby#37659](https://github.com/moby/moby/pull/37659)
+- LCOW: Fix builder using wrong cache layer [moby/moby#37356](https://github.com/moby/moby/pull/37356)
+
 
 ### Logging
 
-* Integration-cli: error logging improvements. [moby/moby#37635](https://github.com/moby/moby/pull/37635)
-* Integration: fix log message. [moby/moby#37542](https://github.com/moby/moby/pull/37542)
-* Loggerutils: fix a typo. [moby/moby#37570](https://github.com/moby/moby/pull/37570)
-* Pass endpoint to the CloudWatch Logs logging driver. [moby/moby#37374](https://github.com/moby/moby/pull/37374)
++ Add "local" log driver [moby/moby#37092](https://github.com/moby/moby/pull/37092)
++ Amazon CloudWatch: add `awslogs-endpoint` logging option [moby/moby#37374](https://github.com/moby/moby/pull/37374)
 * Pass log-level to containerd. [moby/moby#37419](https://github.com/moby/moby/pull/37419)
-+ Add "local" log driver. [moby/moby#37092](https://github.com/moby/moby/pull/37092)
-- Fix logic when enabling buildkit. [moby/moby#37688](https://github.com/moby/moby/pull/37688)
+- Fix json-log file descriptors leaking when using `--follow` [docker/engine#48](https://github.com/docker/engine/pull/48) [moby/moby#37576](https://github.com/moby/moby/pull/37576) [moby/moby#37734](https://github.com/moby/moby/pull/37734)
+- Fix a possible deadlock on closing the watcher on kqueue [moby/moby#37392](https://github.com/moby/moby/pull/37392)
+- Use poller based watcher to work around the file caching issue in Windows [moby/moby#37412](https://github.com/moby/moby/pull/37412)
+
 
 ### Networking
 
++ Add support for global default address pools [moby/moby#37558](https://github.com/moby/moby/pull/37558) [docker/cli#1233](https://github.com/docker/cli/pull/1233)
+* Use direct server return (DSR) in east-west overlay load balancing [docker/engine#93](https://github.com/docker/engine/pull/93) / [docker/libnetwork#2270](https://github.com/docker/libnetwork/pull/2270)
 * Builder: temporarily disable bridge networking when using buildkit. [moby/moby#37691](https://github.com/moby/moby/pull/37691)
-* Move network conversions out of API router. [moby/moby#37156](https://github.com/moby/moby/pull/37156)
-* Use sortorder library for sorting network list output. [docker/cli#1266](https://github.com/docker/cli/pull/1266)
-- Fix faulty error type checking in removeNetwork(). [moby/moby#37409](https://github.com/moby/moby/pull/37409)
+- Handle systemd-resolved case by providing appropriate resolv.conf to networking layer [moby/moby#37485](https://github.com/moby/moby/pull/37485)
```

Comment:
@fcrisciani @ctelfer are there other notable changes (or fixes) in libnetwork between Docker 18.06 and 18.09 ?